sqlgen/expression_sql_generator_test.go

```diff
@@ -565,16 +565,16 @@ func (esgs *expressionSQLGeneratorSuite) TestGenerate_BooleanExpression() {
 			int64(1), int64(2), int64(3),
 		}},
 
-		expressionTestCase{val: ident.In(ae), sql: `("a" IN ((SELECT "id" FROM "test2")))`},
-		expressionTestCase{val: ident.In(ae), sql: `("a" IN ((SELECT "id" FROM "test2")))`, isPrepared: true},
+		expressionTestCase{val: ident.In(ae), sql: `("a" IN (SELECT "id" FROM "test2"))`},
```

Comment:
There is a `expressionTestCase{val: ident.Eq(ae), sql: `("a" IN (SELECT "id" FROM "test2"))`},` above, so maybe this was intended for some reason(??)

But looks strange anyway, I don't think that this should be a desired behaviour